notebooks/model_explorer_streaming.ipynb

```diff
@@ -0,0 +1,1140 @@
+{
```

Comment:
<p>If you like, you could add a line at the top here that says: _by Tilde Thurium: LI &amp; Twitter_ or any social you'd like people to follow you on :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>